src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -152,6 +152,27 @@ RegularAndStaticColumns getFetchedColumns(TableMetadata metadata, RegularAndStat
             {
                 return queried;
             }
+        },
+
+        /**
+         * Fetch only the columns that have are part of the fetched set.
+         *
+         * <p>This strategy is only used to deal with schema propagation delay that caused the replica to have more
+         * columns than the coordinator.</p>
+         */
+        FETCHED_COLUMNS
```

Comment:
nit: personally I find this name a bit confusing - what about something like `FIXED_COLUMNS`, `EXACT_COLUMNS`? I'd just avoid saying "fetched" columns because the strategy is about to determine what the fetched columns are

Reply:
I changed the name.